vslib/src/VirtualSwitchSaiInterface.cpp

```diff
@@ -659,6 +659,12 @@ sai_status_t VirtualSwitchSaiInterface::create(
             return SAI_STATUS_FAILURE;
         }
 
+        //Initialize switch for VOQ attributes
+        if ((ss->initialize_voq_switch_objects(attr_count, attr_list)) != SAI_STATUS_SUCCESS)
```

Comment:
this seems wrong to me, this should not be here, since there can be warm boot executed, and voq objects may be different than cold boot, remove

Reply:
The system ports are one time static configuration done at the start of the system. Unlike m_port_list, no configuration will change system ports and other voq switch objects during lifetime of the system. As of now, irrespective of whether the cold boot or warm boot, the system ports config is required to be always freshly configured. And since attr_count and attr_list are needed to initialize voq switch objects, voq swith object initialization is done from here. In the future, suppose dynamic configuration of system ports are implemented, we can always check for the warmboot state here and take action appropriately.